webapp/components/NWBListViewer.js

```diff
@@ -102,13 +126,24 @@ export default class NWBListViewer extends Component {
   }
 
   render () {
-
-    return <ListViewer
-      columnConfiguration={this.getColumnConfiguration()}
-      instances={this.getInstances()}
-      handler={this}
-      infiniteScroll={true}
-      update={this.state.update} />
+    let instances = this.getInstances();
+    
+    return <div>
+      <ListViewer
+        columnConfiguration={this.getColumnConfiguration()}
+        instances={instances}
+        handler={this}
+        infiniteScroll={true}
+        update={this.state.update} 
+        headingComponent ={
+          <div>
+            <span>
+              {instances.length} Matching Results
+            </span>
+            <span style={{ float : "right" , marginRight : "5vh" }} className="fa fa-area-chart list-icon" onClick={this.plotAllInstances} />
```

Comment:
Better check if there is at least one timeseries in the currently shown isntances, otherwise don't show